<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hotseat_player_count_window</class>
 <widget class="QMainWindow" name="hotseat_player_count_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="menu_frame">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>311</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border-radius: 7px;
	color: rgb(62, 231, 41);
	background-color: rgb(40, 109, 50);
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="name_label">
     <property name="geometry">
      <rect>
       <x>-40</x>
       <y>20</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Scrabble&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_button">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>7</y>
       <width>14</width>
       <height>14</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(255, 40, 54);
	border-radius: 7px;
	border: none
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QFrame" name="game_mode_frame">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>231</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
background-color: rgb(22, 60, 27);
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="game_mode_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(226, 224, 158);

}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Tryb gry&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="hotseat_mode_label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	
	color: rgb(62, 232, 41);
}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700; font-style:italic;&quot;&gt;hot seat&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="game_mode_desc">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	
	color: rgb(204, 204, 143);
}</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:14pt; font-weight:700; font-style:italic;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:13pt;&quot;&gt;w tym trybie rozgrywki zmierzysz się innymi ludźmi&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="player_count_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	
	color: rgb(204, 204, 143);
}</string>
      </property>
      <property name="text">
       <string>wybierz liczbę graczy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="player2_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius: 20;
	background-color: rgb(202, 78, 68);
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="player3_button">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>240</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius: 20;
	background-color: rgb(202, 78, 68);
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="player4_button">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>230</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius: 20;
	background-color: rgb(202, 78, 68);
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QLabel" name="hotseat_icon_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>65</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	
	image: url(:/newPrefix/images/gamer.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="return_to_menu_button">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>450</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: transparent;
image: url(:/newPrefix/images/returnleft.png);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../src_qt.qrc"/>
 </resources>
 <connections/>
</ui>
